webots-docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+ARG BASE_IMAGE=nvidia/cuda:11.8.0-base-ubuntu22.04
+FROM ${BASE_IMAGE} AS downloader
+
+# Determine Webots version to be used and set default argument
+ARG WEBOTS_VERSION=R2022b
+ARG WEBOTS_PACKAGE_PREFIX=
+
+# Disable dpkg/gdebi interactive dialogs
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --yes wget bzip2 && rm -rf /var/lib/apt/lists/ && \
```

Comment:
Personally, I much prefer having each package in an install command in a dockerfile to be on its own line, as well as putting each command on its own line. Makes it much easier to read imo